cypress/integration/pdrs_spec.js

```diff
@@ -223,12 +221,11 @@ describe('Dashboard PDRs Page', () => {
             .should('have.text', `${Number(granule.duration.toFixed(2))}s`);
           cy.get('@columns').eq(5).invoke('text')
             .should('match', /.+ago$/);
-
-          cy.route('DELETE', `/granules/${granule.granuleId}`).as('deleteGranule');
+          cy.intercept('DELETE', `/granules/${granule.granuleId}`).as(`deleteGranule${granule.granuleId}`);
```

Comment:
Also, like above, you can't reassign an alias and then wait for it, so here we use unique aliases.